### ISSUE TITLE
Handle translated fields

### ISSUE THREAD

**Issue by wbobeirne** [wbobeirne/datocms-plugin-nextjs-preview#1]
Need to investigate more, but some folks translate their slugs and need that to be taken into account for the URL.

**Comment by naren-bellala** [comment]
Any plans to include locales in your `getFieldValue` for example: `plugin.getFieldValue('title', 'en')`?

**Comment by wbobeirne** [comment]
Yep, I haven't actually tried out multiple locales in DatoCMS yet so I'm not sure how it's controlled or if I would have to provide a control. I was going to set up a test instance some time soon to try it out with the plugin, but if anyone jumps to it first I'm happy to review any PRs.

**Comment by bramvdpluijm** [comment]
I'm following this issue as well!
